packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -156,5 +161,7 @@ export const projectPaths = {
   sfdxProjectConfig,
   toolsFolder,
   lwcTestResultsFolder,
-  relativeStateFolder
+  relativeStateFolder,
+  relativeToolsFolder,
+  TOOLS
```

Comment:
nit: I'd just export the TOOLS constant on line 17 vs including constants and functions here.